### DISCUSSION TITLE
Ferry Routes show as TravelType : Car

### DISCUSSION THREAD

**Discussion by swapnilmarotidukare-tomtom** [valhalla/valhalla#3639] [General]
Hi,

When I run route api between two points on remote Island with ferry route nearby.
I get route via ferry, but in response its travel_mode = "drive" and travel_type="car",  shouldn't it be transit and ferry?

API JSON body `https://valhalla1.openstreetmap.de//route?json={"costing":"auto","costing_options":{"auto":{"exclude_polygons":[],"maneuver_penalty":5,"country_crossing_penalty":0,"country_crossing_cost":600,"width":1.6,"height":1.9,"use_highways":1,"use_tolls":1,"use_ferry":1,"ferry_cost":300,"use_living_streets":0.5,"use_tracks":0,"private_access_penalty":450,"ignore_closures":false,"closure_factor":9,"service_penalty":15,"service_factor":1,"exclude_unpaved":1,"shortest":false,"exclude_cash_only_tolls":false,"top_speed":140}},"exclude_polygons":[],"locations":[{"lon":128.027315,"lat":-1.715248},{"lon":128.131667,"lat":-1.68366}],"directions_options":{"units":"kilometers"},"id":"valhalla_directions"}`

![Capture](https://user-images.githubusercontent.com/83397615/169578402-c86b0a1c-9167-427e-a657-3d4acd0b3cc1.JPG)

**Comment by kevinkreiser** [comment]
you are correct, we dont differentiate between driving a car on the road or driving your car onto a ferry and riding with it across the water. at the end the route is still using a car it just happens to be on a ferry. i guess your larger point is how can you tell when you are on a ferry other than by the maneuver types? indeed the maneuers are marked with a boolean as to whether they are a ferry we just dont serialize that information out. so at the moment, in the valhalla flavored json api, you can only tell its a very by the ferry_enter and ferry_exit maneuver types.
